jekyll/_ccie/overview.md

```diff
@@ -1,33 +1,52 @@
 
-We are constantly working on making the installation process as smooth as possible and expanding the administrative tooling.  We appreciate your feedback on ways to make CircleCI Enterprise easier and more valuable for you and your team, so please contact us at <enterprise-support@circleci.com> with any suggestions.
+* TOC
+{:toc}
 
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. The CircleCI Enterprise solution is installable inside your private cloud or data center and is free to try for two weeks. CircleCI automates build, test, and deployment of software.
 
-### Installation Steps
+## Basic Features
 
-Once you have all of the prerequisites in place, you can either [follow the detailed
-installation steps for AWS]({{site.baseurl}}/enterprise/aws/), or [for environments not on AWS]({{site.baseurl}}/enterprise/docker-install/).
+After a software repository in GitHub Enterprise is added as a project to the CircleCI Enterprise application, every new commit triggers a build and notification of success or failure through webhooks with integrations for Slack, HipChat, Campfire, Flowdock, or IRC notifications. Code coverage results are available from the details page for any project for which a reporting library is added.
 
+CircleCI may also be configured to deploy code to various environments, including the following:
+
+- AWS CodeDeploy
+- AWS EC2 Container Service (ECS)
+- AWS S3
+- Google Container Engine (GKE)
+- Heroku
+
+Other cloud service deployments can be easily scripted using SSH or by installing the API client of the service with your job configuration.
+
+Contact us at <enterprise-support@circleci.com> to request a trial license.
+
+## Build Environments
+
+CircleCI Enterprise installations may run a single Linux or MacOS
+image for builds. Users of your CircleCI Enterprise installation must use this single contianer image for all of their builds and may **not** choose between Precise and Trusty build container images, or custom built
+images, for example. 
+
+**Note:** If your CircleCI Enterprise installation is not open to the internet, you may
+need to connect to a VPN or take other special steps to access the CircleCI application or SSH builds.
+These steps will vary depending on your network setup.
 
 ## Architecture
 
-At a high level, CircleCI Enterprise has two kinds of instances that it needs in order to run: the services box and builder boxes.
+CircleCI Enterprise requires two types of instances to run: service and builder instances. The service type is a single instance that contains the CircleCI application and the resources that store data and run protocols. Any number of builder instances execute your jobs as defined by the plan for your CircleCI account. Both instance types must access your instance of GitHub Enterprise on the network as illustrated in the following architecture diagram. 
```

Comment:
replaced with this (because I like components as a term to replace box)

CircleCI Enterprise consists of two primary components: Services and Builders. Services run on a single instance that is comprised of the core application, storage, and networking functionality. Any number of Builder instances execute your jobs and communicate back to the Services. Both components must access your instance of GitHub or GitHub Enterprise on the network as illustrated in the following architecture diagram.